civis/cli/__main__.py

```diff
@@ -130,11 +136,12 @@ def param_case_map(param_names):
     return result
 
 
-def invoke(method, path, op, *args, json_output=False, **kwargs):
+def invoke(method, path, op, *args, **kwargs):
```

Comment:
Yeah, it didn't look like it was used anywhere except the tests and that none of this was intended to be really be user-facing.